tests/functionnal/Glpi/Inventory/Assets/Printer.php

```diff
@@ -35,6 +35,8 @@
 
 namespace tests\units\Glpi\Inventory\Asset;
 
+use Toolbox;
```

Comment:
All call are done using the full name, this one should be removed ;)

Reply:
done =)